.gitlab-ci.yml

```diff
@@ -100,11 +99,6 @@ package:windows-amd64:
   tags: [go]
   script: go run mage.go -v PackageWindowsAmd64
 
-package:ios:
```

Comment:
As we don't have ios for now there is no point to package it.

Reply:
:rage: